osquery/tables/applications/posix/docker.cpp

```diff
@@ -578,26 +589,42 @@ QueryData genContainerProcesses(QueryContext& context) {
         std::vector<std::string> vector;
         BOOST_FOREACH (const auto& v, processes.second) {
           vector.push_back(v.second.data());
-        }
-
-        // On OS X, ps_args are not being honored. Check number of columns
-        if (vector.size() != 21) {
-          continue;
+          VLOG(1) << v.second.data() << "\n";
         }
 
         Row r;
         r["id"] = id;
         r["pid"] = BIGINT(vector.at(0));
-        r["name"] = vector.at(19);
-        r["cmdline"] = vector.at(20);
+        r["wired_size"] = BIGINT(0); // No support for unpagable counters
```

Comment:
Ignore my previous comment. I didn't see this here.

Reply:
no worries :)